tests/rest/media/v1/test_url_preview.py

```diff
@@ -562,3 +565,126 @@ def test_accept_language_config_option(self):
             ),
             server.data,
         )
+
+    def test_oembed_url(self):
+        """Test changing a URL via oEmbed."""
```

Comment:
"changing"?

Reply:
I updated the comments and names of both of the new tests!